docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+    - MSTS
+    - SSTS
+    - SCTS
+activity:
+  setting:
+    - SetParameterFromExternal
+  validation:
+    - ValidateParameterByExpert
+source:
+  - Initial instrument setup
+simulation_software:
+  - name: sim_telarray
+```
+
+### Valid Keys
+
+Valid keys are described in detail in [model_parameter_and_data_schema.metaschema.yml](https://github.com/gammasim/simtools/blob/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml). The most important parameters are:
+
+#### Header section
+
+- `title`: Title of the schema file.
+- `version`: Version of this schema file.
+- `meta_schema`: Name of the base schema.
+- `meta_schema_url`: URL of the base schema.
+- `meta_schema_version`: Version of the base schema.
+
+#### Parameter description
+
+- `name`: Name of the parameter.
+- `description`: Description of the parameter.
+- `short_description`: Short description of the parameter (optional).
+
+#### Parameter data
+
+The `data` field is used to describe the actual type, format, and allowed values of the parameter.
+
+- `type`: Data type of the parameter.
+- `unit`: Units of the parameter (compatible with astropy units). Note that units are explicitly listed in the jsonschema.yml file.
+- `default`: Default value of the parameter.
+- `allowed_range`: Allowed range of the parameter.
+- `allowed_values`: Allowed values of the parameter. Use if values are not continuous and restricted to a small list of allowed values.
+
+Tables of any dimensions can be described using the `data_table` field. Example:
+
+```yaml
+data:
+  - type: data_table
+    table_columns:
+      - name: wavelength
+        description: Wavelength.
+        required: true
+        unit: nm
+        type: double
+        required_range:
+          min: 300.0
+          max: 700.0
+        input_processing:
+          - remove_duplicates
+          - sort
+      - name: photo_detection_efficiency
+        description: Average quantum or photon detection efficiency.
+        required: true
+        unit: dimensionless
+        type: double
+        allowed_range:
+          min: 0.0
+          max: 1.0
+```
+
+Input processing in form of sorting, removing of duplicates, etc. can be specified using the `input_processing` field.
+
+:::{Warning}
+The `data_table` field is not yet implemented in the simtools schema and parameter files.
+:::
+
+#### Instrument description
+
+- `class`: Instrument class. Allow values are *Camera*, *Site*, *Structure*, *Telescope*
+- `type`: Instrument type following CTAO Naming.
+- `site`: CTAO site. Allowed values are *North* and *South*.
+
+### Activity description
+
+Describes setting and validation activities. Each activity corresponds to a workflow as described in the [simtools workflows repository](https://github.com/gammasim/workflows).
+
+:::{Warning}
+The implementation of activities and workflows is incomplete and in progress.
+:::
+
+### Data source description
+
+Describes the source of the data or parameter (e.g., *Calibration*)
```

Comment:
are the available choices here pre-defined or free?

Reply:
Everything is pre-defined in schemas. I thought this is clear.

But we don't use the data source for anything, and I am not sure if this will be dropped in future.